src/lib/components/ConfirmTransactions.js

```diff
@@ -33,31 +35,166 @@ export default function ConfirmTransactions(props) {
+
+                  {action.type === "Stake" && (
+                    <>
+                      <div>
+                        <span className="text-secondary">Public key: </span>
+                        <span className="font-monospace">
+                          {action.params.publicKey}
+                        </span>
+                      </div>
+                      <div>
+                        <span className="text-secondary">Amount: </span>
+                        <span className="font-monospace">
+                          {formatNearAmount(action.params.stake)} NEAR
+                        </span>
+                      </div>
+                    </>
+                  )}
+
+                  {action.type === "DeployContract" && (
+                    <>
+                      <div>
+                        <span className="text-secondary">Code:</span>
+                        <span className="font-monospace">
+                          {action.params.code}
+                        </span>
+                      </div>
+                    </>
+                  )}
+
+                  {action.type === "DeleteAccount" && (
+                    <>
+                      <div>
+                        <span className="text-secondary">Beneficiary id:</span>
+                        <span className="font-monospace">
+                          {action.params.beneficiaryId}
+                        </span>
+                      </div>
+                    </>
+                  )}
+
+                  {action.type === "Transfer" && (
+                    <>
+                      <div>
+                        <span className="text-secondary">Receiver: </span>
+                        <span className="font-monospace">
+                          {transaction.receiverId}
+                        </span>
+                      </div>
+                      <div>
+                        <span className="text-secondary">Amount: </span>
+                        <span className="font-monospace">
+                          {formatNearAmount(action.params.deposit)} NEAR
+                        </span>
+                      </div>
+                    </>
+                  )}
+
+                  {action.type === "AddKey" && (
+                    <>
+                      <div>
+                        <span className="text-secondary">Public key: </span>
+                        <span className="font-monospace">
+                          {action.params.publicKey}
+                        </span>
+                      </div>
+
+                      {action.params.accessKey.permission === "FullAccess" ? (
+                        <div>
+                          <span className="font-weight-bold">
+                            Carefully! You add a full access key; with this key
+                            you can sign any transaction on behalf of your
+                            account. The key has full access to your tokens.
```

Comment:
I think we should display the warning message at the top of the confirmation window.

Also display a warning message for all non default actions. E.g. contract deploy on your account, all key actions, account deletion.